stable/rabbitmq-ha/README.md

```diff
@@ -256,7 +257,15 @@ the following keys:
 
 ### Prometheus Monitoring & Alerts
 
-Prometheus and its features can be enabled by setting `prometheus.enabled` to `true`.  See values.yaml for more details and configuration options
+As of RabbitMQ 3.8.0, it is possible to enable Prometheus metrics natively, no need to run an external exporter. To enable native Prometheus metrics, set `rabbitmqPrometheusPlugin.enabled` to `true`. This will expose all RabbitMQ node metrics via the `<<rabbitmqhost>>:15692/metrics` URL. Since all metrics are node local, they add the least pressure on RabbitMQ and will be available for as long as RabbitMQ is running, regardless of inter-node pressure or other nodes in the cluster going away.
+
+To learn more about RabbitMQ's native support for Prometheus, please refer to the official [Monitoring with Prometheus & Grafana guide](https://www.rabbitmq.com/prometheus.html).
+
+Team RabbitMQ manages Grafana dashboards that are meant to be used with the native Prometheus support. They are publicly available at [grafana.com/orgs/rabbitmq](https://grafana.com/orgs/rabbitmq).
+
+To enable metrics via the traditional rabbitmq_exporter, set `prometheus.enabled` to `true`. See `values.yaml` file for more details and configuration options.
```

Comment:
`traditional rabbitmq_exporter sidecar container`. Also make rabbitmq_exporter a link to https://github.com/kbudde/rabbitmq_exporter